programmers/난이도별/level01.키패드_누르기/sokkanji.js

```diff
@@ -0,0 +1,63 @@
+function getDistance(target, handLocation) {
+  // 행의 인덱스 거리 + 열의 인덱스 거리
+  const distance = Math.abs(target[0] - handLocation[0]) + Math.abs(target[1] - handLocation[1]);
+
+  return distance;
+}
+
+function solution(numbers, hand) {
+  const keys = {
+    1: [0, 0],
+    2: [0, 1],
+    3: [0, 2],
+    4: [1, 0],
+    5: [1, 1],
+    6: [1, 2],
+    7: [2, 0],
+    8: [2, 1],
+    9: [2, 2],
+    "*": [3, 0],
+    0: [3, 1],
+    "#": [3, 2],
+  };
+  let answer = "",
+    leftHandLocation = keys["*"],
+    rightHandLocation = keys["#"];
+
+  numbers.map((value, index) => {
+    let isLeft = false;
+    switch (value) {
+      case 1:
+      case 4:
+      case 7:
+        isLeft = true;
+        break;
+      case 2:
+      case 5:
+      case 8:
+      case 0:
+        // 누를 키값, 현재 손가락의 위치
```

Comment:
요 부분이 인상깊어서 찾아봤는데 [자바스크립트 MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch#methods_for_multi-criteria_case)에서도 사용하는 것 같네요! 

하지만 [Eric Kent의 의견](https://stackoverflow.com/questions/13569924/best-practice-with-javascript-switch-statement)도 옳은 것 같아 주석을 추가하면 좋을 것 같습니다.

> The best practice is to avoid repeating code. 
Otherwise, if the code is later edited, someone else might not realise that it is intentional, and add the break in where it is missing.
(다른 사람이 의도한 코드인 줄 모르고 break를 추가할 수도 있다는 이야기입니다.)

Reply:
리뷰 해주신 대로 주석 추가해서 commit 했습니다!! 피드백 감사합니다😄